whole-repo-tests/test_mypy.py

```diff
@@ -16,6 +16,8 @@
 from hypothesistooling.projects.hypothesispython import PYTHON_SRC
 from hypothesistooling.scripts import pip_tool, tool_path
 
+PYTHON_VERSIONS = [f"3.{v}" for v in range(7, 11)]
```

Comment:
I think I'd prefer to write these out as literals, unless we can pull them out of the autoupdated CI config? Just thinking about how they'll stay up to date. I think we can also test against 3.11?